src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -198,10 +198,7 @@ describe('ReactDOMComponent', function() {
 
       expect(console.error.argsForCall.length).toBe(1);
       expect(console.error.argsForCall[0][0]).toEqual(
-        'Warning: `span` was passed a style object that has previously been ' +
-        'mutated. Mutating `style` is deprecated. Consider cloning it ' +
-        'beforehand. Check the `render` using <span>. Previous style: ' +
-        '{fontSize: NaN}. Mutated style: {fontSize: NaN}.'
+        'Warning: `NaN` is an invalid value for the `fontSize` css style property'
```

Comment:
We should be probably just remove this test and write a new one in CSSPropertyOperations-test (since we are now testing that unit not ReactDOMComponent - this one is just surfacing the warning that was emitted before)